main_test.go

```diff
@@ -21,294 +21,183 @@
 package main
 
 import (
-	"errors"
+	"bufio"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
 	"testing"
 
-	"github.com/codegangsta/cli"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/mock"
+	gflags "github.com/jessevdk/go-flags"
 )
 
-func TestCreateAndRunApp(t *testing.T) {
-	mockCommander := new(mockCommander)
-	torcher := &torcher{
-		commander: mockCommander,
-	}
+const testPProfInputFile = "./pprof/testdata/pprof.1.pb.gz"
 
-	var validateContext = func(args mock.Arguments) {
-		context := args.Get(0).(*cli.Context)
-		assert.NotNil(t, context)
-		assert.Equal(t, "go-torch", context.App.Name)
+func getDefaultOptions() *options {
+	opts := &options{}
```

Comment:
I don't think there's any strong preferences - `new` was kind of the predecessor to the `&T{}` syntax. There's a few online discussions about wanting to remove `new` but it'll stay because of backwards compatibility.

I tend to prefer `&T{}`